src/main/perl/ComponentProxy.pm

```diff
@@ -336,7 +337,7 @@ sub _setDependencies {
             push (@{$self->{'PRE_DEPS'}},$el->getStringValue());
         }
         $self->debug(2, "pre dependencies for component $self->{'NAME'}: ",
-                     ' '.join(',',@{$self->{PRE_DEPS}}));
+                     join(',',@{$self->{PRE_DEPS}}));
```

Comment:
not really needed no, but it's annoying and wrong (and this is debug level 2, not really error messages)